.gitignore

```diff
@@ -5,3 +5,4 @@ source/*/*.min.js.map
 test/results
 /dev.sh
 .DS_Store
+.idea
```

Comment:
My project settings are stored with each specific project as a set of xml files under the  [.idea](https://www.jetbrains.com/idea/) folder.